Makefile

```diff
@@ -0,0 +1,62 @@
+all: lint test check-coverage build  ## build, lint, test and check coverage
+	@[ -e .git/rebase-merge ] && git --no-pager log -1 --pretty='%h %s'
```

Comment:
`@[ ! -e .git/rebase-merge ] || git --no-pager log -1 --pretty='%h %s'`

Reply:
Done. Changed to `if` as discussed in person.

Reply:
Done in 6b86995d99b025b7e006e9aca4fd18187b62bc68